src/View.php

```diff
@@ -197,12 +197,15 @@ public function setModel(\atk4\data\Model $m)
     /**
      * Sets source of the View.
      *
-     * @param array $data
+     * @param array $data   Array of data
+     * @param array $fields Limit model to particular fields
      *
      * @return \atk4\data\Model
      */
-    public function setSource(array $data)
+    public function setSource(array $data, $fields = null)
     {
+        $this->setModel(new \atk4\data\Model(new \atk4\data\Persistence_Static($data)), $fields);
+        /*
```

Comment:
Should we leave all that uncommented block here?

Reply:
I'll remove.